From 441d29bb0225000d840d4fe8c67e5461e40ab101 Mon Sep 17 00:00:00 2001
From: Karl Berry <karl@freefriends.org>
Date: Sun, 19 Jan 2020 18:24:28 -0800
Subject: [PATCH 24/25] lint: make syntax-check same as maintainer-check.

* maintainer/syntax-checks.mk (syntax-check): new target,
same as maintainer-check. For consistency with many other
GNU packages.
(.PHONY): add here too.
---
 maintainer/syntax-checks.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/maintainer/syntax-checks.mk b/maintainer/syntax-checks.mk
index 91cf92475..3b32950e4 100644
--- a/maintainer/syntax-checks.mk
+++ b/maintainer/syntax-checks.mk
@@ -80,8 +80,8 @@ sc_tabs_in_texi \
 sc_at_in_texi
 
 $(syntax_check_rules): bin/automake bin/aclocal
-maintainer-check: $(syntax_check_rules)
-.PHONY: maintainer-check $(syntax_check_rules)
+maintainer-check syntax-check: $(syntax_check_rules)
+.PHONY: maintainer-check syntax-check $(syntax_check_rules)
 
 # Check that the list of tests given in the Makefile is equal to the
 # list of all test scripts in the Automake testsuite.
-- 
2.25.0

